test/games/prime/patcher_file_lib/test_pickup_exporter.py

```diff
@@ -472,8 +472,12 @@ def test_pickup_scan_for_progressive_suit(echoes_pickup_database, echoes_resourc
 @pytest.mark.parametrize(
     ("item", "ammo", "expected"),
     [
-        ("Beam Ammo Expansion", [4, 20], "Provides 4 Dark Ammo, 20 Light Ammo and 1 Item Percentage."),
-        ("Missile Expansion", [4], "Provides 4 Missiles and 1 Item Percentage."),
+        (
+            "Beam Ammo Expansion",
+            [4, 20],
+            "Provides 4 Dark Ammo, provides 20 Light Ammo and provides 1 Item Percentage.",
```

Comment:
hmm, so if one has positive dark ammo but negative light ammo, then this check would have to be `Provides 4 Dark Ammo, 1 Item Percentage and removes 1 Light Ammo`? I personally feel like that's less readable than how it is above....

Reply:
The item percentage thing is a fancy thing that could go away but is more effort.
As for readability, the ingame font size is quite big. I'm actually concerned this new text is too big to fit in one box

Reply:
I've been thinking about this more, this maybe should be redone a bit differntly?
if you have negative dark ammo, negative light ammo, positive item percentage, positive foo and 1 negative bar, then I feel like these should be split.
So instead of `Provides 1 Item percentage, 1 foo, removes 4 Dark Ammo, 1 Light Ammo and 1 bar`, have something like this instead `Provides 1 Item Percentage and 1 foo. Removes 4 Dark Ammo, 1 Light Ammo and 1 bar.`
Thoughts?

Reply:
Seems good.

Reply:
This has been addressed